docs/cirq_interface.md

```diff
@@ -148,7 +148,26 @@ matrices, if one is specified.
 
 ### Parametrized circuits
 
-QSimCircuit objects can also contain
+`QSimCircuit` objects can also contain
 [parameterized gates](https://cirq.readthedocs.io/en/stable/docs/tutorials/basics.html#Using-parameter-sweeps)
 which have values assigned by Cirq's `ParamResolver`. See the link above for
 details on how to use this feature.
+
+### GPU execution
+
+`QSimSimulator` provides optional support for GPU execution of circuits, which
+may improve performance. In order to use this feature, qsim must be compiled on
+a device with the [CUDA toolkit](https://developer.nvidia.com/cuda-downloads)
+and run on a device with available NVIDIA GPUs.
+
+Compilation for GPU follows the same steps outlined in the
+[Compiling qsimcirq](./cirq_interface.md#compiling-qsimcirq) section.
+
+`QSimOptions` provides four parameters to configure GPU execution:
+* use_gpu: whether to use GPU instead of CPU for simulation.
+* gpu_sim_threads: number of threads per CUDA block to use for the GPU
```

Comment:
Could you clarify that the last three parameters are for performance fine-tuning and the user does not need to specify them as the default values are good enough in most cases?

Reply:
Done.